packages/components/psammead-inline-link/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-inline-link",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Please update the `package-lock.json` too.

Reply:
The comment under the `roadmap` can likely be removed, too: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-inline-link/README.md#roadmap